static/app/components/globalAppStoreConnectUpdateAlert/updateAlert.tsx

```diff
@@ -1,67 +1,277 @@
-import {useContext} from 'react';
+import {Fragment, useContext, useEffect, useState} from 'react';
 import styled from '@emotion/styled';
+import find from 'lodash/find';
+import mapValues from 'lodash/mapValues';
 
+import {Client} from 'app/api';
 import Alert from 'app/components/alert';
+import Button from 'app/components/button';
+import Link from 'app/components/links/link';
 import AppStoreConnectContext from 'app/components/projects/appStoreConnectContext';
-import {IconRefresh} from 'app/icons';
+import {IconClose, IconRefresh} from 'app/icons';
+import {t} from 'app/locale';
 import space from 'app/styles/space';
 import {Organization, Project} from 'app/types';
+import {AppStoreConnectStatusData} from 'app/types/debugFiles';
+import {promptIsDismissed} from 'app/utils/promptIsDismissed';
+import withApi from 'app/utils/withApi';
+
+const APP_STORE_CONNECT_UPDATES = 'app_store_connect_updates';
 
 type Props = {
+  api: Client;
   organization: Organization;
   project?: Project;
   Wrapper?: React.ComponentType;
   isCompact?: boolean;
   className?: string;
 };
 
-function UpdateAlert({Wrapper, project, className}: Props) {
+function UpdateAlert({api, Wrapper, isCompact, project, organization, className}: Props) {
   const appStoreConnectContext = useContext(AppStoreConnectContext);
+  const [isDismissed, setIsDismissed] = useState(false);
+  const [firstError, setFirstError] = useState(
```

Comment:
not sure about this... shouldn't we display warnings for each app store connect repository?... it can be that the errors are different

Reply:
sorry about that, i neglected to explain what was going on in this implementation 🙃 i focused on just trying to make sure one alert was shown so i could test that the backend was working from end to end.

this is also compounded by the fact that the backend work most likely needs a design review as i'm not too confident in the implementation, so some throwaway basic "just works" code here was rigged up.